src/bindings/python/src/pyopenvino/core/async_infer_queue.cpp

```diff
@@ -21,71 +21,85 @@ namespace py = pybind11;
 
 class AsyncInferQueue {
 public:
-    AsyncInferQueue(std::vector<InferRequestWrapper> requests,
-                    std::queue<size_t> idle_handles,
-                    std::vector<py::object> user_ids)
-        : _requests(requests),
-          _idle_handles(idle_handles),
-          _user_ids(user_ids) {
+    AsyncInferQueue(ov::CompiledModel& model, size_t jobs) {
```

Comment:
Does `AsyncInferQueue` modify the model? If not it should be `const &`.

Reply:
It is not possible to use `create_infer_request()` on `const ov::CompiledModel` with current API:
https://github.com/openvinotoolkit/openvino/blob/7019afd7ff5be2e9798c2fcf2e9a7e18eb8be1c7/src/inference/include/openvino/runtime/compiled_model.hpp#L146-L152